f/briefing/fn_createBriefing.sqf

```diff
@@ -72,18 +72,37 @@ if (_unitfaction in ["opf_f","opf_t_f"]) exitwith {
 	[_unitfaction] call _fnc_debug;
 };
 
+// OPFOR > Spetsnaz
+if (_unitfaction in ["opf_r_f"]) exitwith {
+	#include "f_briefing_spetsnaz.sqf"
+	[_unitfaction] call _fnc_debug;
+};
+
 // INDEPENDENT > AAF
 if (_unitfaction in ["ind_f"]) exitwith {
 	#include "f_briefing_aaf.sqf"
 	[_unitfaction] call _fnc_debug;
 };
 
+// INDEPENDENT > LDF
+if (_unitfaction in ["ind_e_f"]) exitwith {
+	#include "f_briefing_ldf.sqf"
+	[_unitfaction] call _fnc_debug;
+};
+
 // INDEPENDENT > SYNDIKAT
 if (_unitfaction in ["ind_c_f"]) exitwith {
 	#include "f_briefing_syndikat.sqf"
 	[_unitfaction] call _fnc_debug;
 };
 
+// INDEPENDENT > Looters
```

Comment:
Might be good to say 'NPR (Looters)' to make the distinction between our name and the ingame name clear